_posts/2017-07-23-rustdaymx-2017.md

```diff
@@ -0,0 +1,64 @@
+
+Finally, a third session was held to go through the _Web Scraper_ and _Text Adventure_ modifications, push it to [GitHub](https://github.com/rustmx) so it was available for people to fetch it, and make sure the mentors were prepared to guide the attendees.
+
+## The day of the event
+
+The event day arrived and everyone was really anxious about it. We arrived the venue early to set up everything. People started coming and the _Installers_ team was there to help them. _Brian_ and _Erick_ were also helping everyone have their environments set up.
+
+We started with some quick welcome words, highlighting the different teams available for attendees and quickly passing the baton to _Erick_ for the [keynote](https://youtu.be/Wq_Mlv1-mSI), a detailed introduction to the language.
+
+Then, the first workshop was done as an [Introduction to the Rust Ecosystem](https://youtu.be/hZ8DGnTi254), by _Norberto Ortigoza_, followed by a lighting talk by _Zura Guerra_ talking about her experience with Rust in local hackathons.
+
+Next, _Erich Córdoba_ talked about [How to convince your boss to use Rust](https://youtu.be/1EFioxmCcic), and after that we had some [delicious basket tacos](https://flic.kr/p/VX7qYr).
+
+After the lunch, _Abiel Parra_ did the second workshop to show the attendees how to [Implement a Cryptocoin Prices Scraper using Rust](https://youtu.be/APBJmZ-dRbA), so they could start going a little bit deeper into the Rust syntax and learn how to use crates.
+
+To finish, _Sebastián Magrí_ demonstrated how to [Build a Text Adventure](https://youtu.be/i1wU70E1yS8), focused on explaining project organization, data structures and lifetimes.
+
+## Wrap up
+
+In the end, the team and the attendants were really satisfied. There were no CoC related issues and we got a peak of 52 participants that, plus volunteers and staff, did a total number of 72 persons.
+
+The code of the workshops is available at [Rust-MX's GitHub](https://github.com/rustmx), and 3 weeks later, the team had the opportunity to follow up with two workshops at [TelmexHub](). The next meetup is set to happen on July 26th, and we're already planning another whole day event for November or December. We had a lot of lessons learned from this first time, which we look forward to apply to the next one.
+
+We hope this experience is useful for people in other places to do their own workshop days. Please, get in touch with [us](mailto:community-team@rust-lang.org) if you need support or feedback. We look forward to work with you to bring more Rust to more people.
```

Comment:
Put `get in touch with us` into the mailto link (instead of just the single word). Much easier to hit then.